pages/index.js

```diff
@@ -70,25 +29,25 @@ export default function Landingpage() {
   };
   return (
     <>
-      <Header>
-        <h1>Travel Journal</h1>
-      </Header>
-      <Form onSubmit={handleSubmit}>
-        <Fieldset>
-          <Legend>Where is your journey going?</Legend>
-          <Label htmlFor="text">City or Country?</Label>
-          <Input
-            type="text"
-            required
-            id="text"
-            name="text"
-            placeholder="enter your destination..."
-          />
-          <Button type="submit">Submit</Button>
-          <p>{output}</p>
-        </Fieldset>
-      </Form>
-      <Link href="/EntryListPage">Your Entries</Link>
+      <main>
+        <form onSubmit={handleSubmit}>
+          <StyledFieldset>
+            <StyledLegend>Where is your journey going?</StyledLegend>
+            <StyledLabel htmlFor="text">City or Country</StyledLabel>
+            <input
+              type="text"
+              required
+              id="text"
+              name="text"
+              pattern="^[^\sa0-9].*$"
+              maxLength="15"
+              placeholder="enter your destination..."
```

Comment:
setting a placeholder tends to harm a11y.  Either the contrast is too low, or it is assumed that the field has already been filled in by the placeholder. Therefore, I tend to advise against the use of placeholders.

Reply:
I saw that you used my pattern. There's a typo in, the 'a' is wrong (because it excludes an 'a' as first letter) ;-)

Reply:
Yeah, right! Nothing what i found online worked out, so i tried yours..Thanks for letting me know :)